examples/02_second_level_models/plot_11_second_level_simple_t_test_calculations.py

```diff
@@ -0,0 +1,68 @@
+"""
+Second level analysis: one sample t-test
+========================================
+
+Perform a one-sample t-test on a bunch of images 
+(a.k.a. second-level analyis in fMRI) and threshold a statistical image.
+This is based on the so-called localizer dataset.
+It shows activation related to a mental computation task,
+as opposed to narrative sentence reading/listening.
+
+Author: Bertrand.thirion, Virgile Fritsch, 2014--2015
+
+"""
+
+#########################################################################
+# Prepare some images for a simple t test
+# ----------------------------------------
+# This is a simple manually performed second level analysis
+n_subjects = 20
+
+from nilearn import datasets
+localizer_dataset = datasets.fetch_localizer_calculation_task(
+    n_subjects=n_subjects)
+
+# mask data
+from nilearn.input_data import NiftiMasker
+nifti_masker = NiftiMasker(
+    smoothing_fwhm=5,
+    memory='nilearn_cache', memory_level=1)  # cache options
+cmap_filenames = localizer_dataset.cmaps
+fmri_masked = nifti_masker.fit_transform(cmap_filenames)
+
+#########################################################################
+# Perform the second level analysis
+# ----------------------------------
+# perform a one-sample test on these values
+from scipy.stats import ttest_1samp
```

Comment:
This part of the example should be rewritten to use Nistats Second Level model instead of the low-level scipy function.